src/side-navigation/side-navigation.scss

```diff
@@ -1,205 +1,144 @@
 // Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 // See LICENSE.md in the project root for license terms and full copyright notice.
 @import '../style/index';
-@import '../icons/index';
+@import '../icon/index';
 
 @mixin iui-side-navigation {
   display: inline-flex;
   flex: 0 0 auto;
   flex-direction: column;
-  overflow: hidden;
   transition: max-width $iui-speed-fast ease-out;
   height: 100%;
-
   @include themed {
-    background-color: t(iui-color-background-1);
-    border: 1px solid t(iui-color-background-5);
+    background-color: t(iui-color-background-3);
+    border-right: 1px solid t(iui-color-background-5);
   }
 
-  > .iui-expand-container {
-    height: $iui-l;
-    cursor: pointer;
-    display: inline-flex;
-    justify-content: center;
-    align-items: center;
-
-    @include themed {
-      background-color: t(iui-color-background-1);
-    }
-
-    > .iui-icon {
-      width: $iui-icons-small;
-      height: $iui-icons-small;
-
-      @include themed {
-        fill: t(iui-icons-color);
-      }
-    }
-
-    &:hover > .iui-icon {
-      @include themed {
-        fill: t(iui-icons-color-primary);
-      }
-    }
-
-    &:focus {
-      outline: none;
-    }
-
-    &:focus-visible {
-      @include themed {
-        box-shadow: inset 0 0 $iui-xs
-          rgba(t(iui-color-foreground-primary-rgb), t(iui-opacity-3));
-      }
-    }
-  }
-
-  > .iui-sidenav-content {
+  > .iui-content {
     display: flex;
     flex-direction: row;
-    height: calc(100% - #{$iui-l});
+    height: calc(100% - #{$iui-line-height});
 
-    > .iui-sidenav-items-container {
+    > .iui-content-container {
       display: flex;
       flex-direction: column;
+      width: 100%;
```

Comment:
What it gives?

Reply:
Without it the buttons do not shrink to fit within the collapsed side nav.  Changing the width helps with `focus` in the collapsed mode.
![Screen Shot 2021-05-13 at 10 35 08 AM](https://user-images.githubusercontent.com/849817/118140991-c037b800-b3bd-11eb-9b50-64697138ca94.png)

Reply:
After my changes this seems to do nothing now 😀 but maybe I am missing something.

Reply:
After commenting out `width: 100%` the Administration button was still having the same issue as above.  I added some CSS to handle the bottom section (lines 23-40) and everything seems to work correctly now.